scripts/g.extension/testsuite/test_addons_download.py

```diff
@@ -120,7 +120,9 @@ def test_github_install_official(self):
 
         for file in files:
             self.assertFileExists(file)
-            if file.suffix != ".html":
+            if file.suffix != ".html" and not ms_windows:
+                # Prebuild C-modules for MS Windows are unlikely to run
+                # on fresh compiled GRASS GIS
```

Comment:
I thought this should fail at g.extension step already.

Reply:
Addons are pre-build for MS Windows also for the development version:
https://wingrass.fsv.cvut.cz/grass83/addons/grass-8.3.dev/logs/
However, getting the addon-help fails because build versions differ...

Reply:
Oh, I missed what test_github_install_official is. That makes sense. Mention the different build versions in the comment.